### DISCUSSION TITLE
Feature to search within blog content

### DISCUSSION THREAD

**Discussion by cutbypham** [satnaing/astro-paper#336] [Ideas]
I like how Astro Cactus uses Pagefind for in-content search. Can Fuse.js do something similar?
![image](https://github.com/user-attachments/assets/1553b5a4-9346-43f2-8a93-7a5cdaa8426e)
https://astro-cactus.chriswilliams.dev/

**Comment by satnaing** [comment]
I believe Fuse.js alone could not perform such feature.
I'll look into `Pagefind` for search feature in AstroPaper.

**Reply by linwanxiaoyehua** [reply]
> I believe Fuse.js alone could not perform such feature. I'll look into `Pagefind` for search feature in AstroPaper.

By the way, Is there any way to move TOC to the right side instead of top of post, like this one below:

<img width="1266" alt="image" src="https://github.com/user-attachments/assets/ac414700-e049-400b-aca4-88d86b648588">

**Comment by satnaing** [comment]
AstroPaper v5 uses Pagefind as static search so this is possible now.

![AstroPaper v5 Search](https://github.com/user-attachments/assets/a09f1ffc-60bd-4f73-8667-77078508a05d)

https://astro-paper.pages.dev/search/
